timescale-container/quantumleap-db-setup.py

```diff
@@ -436,20 +436,25 @@ def run(self):
 
 
 def run():
-    sleep(5)  # Give Postgres enough time to start in the container.
-    try:
-        args = Args().get()
-        DbBootstrapCmd(args).run()
-        DbInitCmd(args).run()
-        DbLoadCmd(args).run()
-    except CalledProcessError as cpe:
-        # Rewrite error message to avoid leaking passwords into log files.
-        msg = 'Command `{0}` did not complete successfully. Exit status: {1}' \
-            .format(cpe.cmd[0], cpe.returncode)
-        print(msg, file=sys.stderr)
-        if cpe.output is not None:
-            print(str(cpe.output), file=sys.stderr)
-
+    count = 0
+    retry = True
+    while retry and count < 10:
+        try:
```

Comment:
@c0c0n3 sometimes it happens that the setup is launched when timescale is not yet ready, a retry loop helps with that